tests/run_tests.py

```diff
@@ -80,7 +80,7 @@ def setup_suite(self, test_names):
 
     def run_tests(self, test_names):
         self.setup_suite(test_names)
-        return unittest.TextTestRunner(verbosity=2).run(self.suite)
+        return unittest.TextTestRunner(stream=sys.stdout, verbosity=2).run(self.suite)
```

Comment:
Having the test runner always write to stdout makes it a lot easier to redirect everything to a file without missing anything.

Reply:
As opposed to writing to stderr?

Reply:
Right now the tests are outputting to a mixture of stdout and stderr, so this simply makes sure everything is sent to stdout. We have some print statements where there are errors sometimes. I understand the usefulness of using stderr, but right now the tests are inconsistent which means we only get a cohesive story when everything goes to the same stream.